### ISSUE TITLE
When Querying Tally ERP Through ODBC Certain Fields Cause A Wrong Column Error

### ISSUE THREAD

**Issue by azwalzaman** [alexbrainman/odbc#200]
So this works perfectly for most queries with Tally but for certain fields I get this error `wrong column #0 length 0 returned, 8 expected`. Same code works for say `SELECT $Name FROM StockItem` and the query that causes errors here works fine with PyODBC. Any idea what I can do for this? The value of the field when extracted through PyODBC is a string of the form `Price/Unit` for example `99.00/nos`

```
import (
	"database/sql"
	"fmt"
	"log"
)

func ReadProducts(db *sql.DB) {
	// Start with a simplified query
	rows, err := db.Query("SELECT $StandardPrice FROM StockItem")
	if err != nil {
		log.Fatal("Failed to execute query:", err)
	}
	defer rows.Close()

	for rows.Next() {
		var stockItemName sql.NullString

		err := rows.Scan(&stockItemName)
		if err != nil {
			log.Fatal("Failed to scan row:", err)
		}

		fmt.Printf("Stock Item Name: %s\n", stockItemName.String)
	}

	if err = rows.Err(); err != nil {
		log.Fatal("Error occurred during row iteration:", err)
	}
}
```

**Comment by alexbrainman** [comment]
No idea.

I used this driver to connect to MS SQL Server. Never to TallyODBC64_9000 . Maybe you can adjust the code so it works with TallyODBC64_9000 .

Alex

**Comment by azwalzaman** [comment]
> No idea.
> 
> I used this driver to connect to MS SQL Server. Never to TallyODBC64_9000 . Maybe you can adjust the code so it works with TallyODBC64_9000 .
> 
> Alex

Yeah, fair. I was considering it already actually but I wanted to see if you have any idea of what it could be first

**Comment by alexbrainman** [comment]
> I was considering it already actually but I wanted to see if you have any idea of what it could be first.

No idea. But all source code is there.

Alex

**Comment by webmotive1010** [comment]
@azwalzaman Facing same issue while fetching form Tally, string type fields works fine but number like $OpeningBalance, $ClosingBlance throws runtime error "wrong column #0 length 0 returned, 8 expected". Any luck?
